src/edu/psu/sweng500/team8/play/SudokuAction.java

```diff
@@ -14,12 +15,14 @@
 	private CellCoordinates cellCordinates;
 	private int previousValue;
 	private int newValue;
+	private PencilMarkManager pencilMarkManager; 
 
 	public SudokuAction(CellCoordinates cellCoordinates,

```

Comment:
I meant keep the Cell in place of CellCoordinates. That way you don't have to look it up from the Board every time.